src/Microsoft.Health.Fhir.TemplateManagement/Models/DefaultTemplateInfo.cs

```diff
@@ -18,16 +18,24 @@ public DefaultTemplateInfo(DataType dataType, string imageReference, string temp
             TemplatePath = templatePath;
         }
 
+        public static string Hl7v2DefaultTemplatesResource => "Hl7v2DefaultTemplates.tar.gz";
```

Comment:
Not sure I follow.  Is the ask to ensure there is a package for this code that has the default templates embedded as resources or are you suggesting the default templates be packaged in a nuget in a manner besides an embedded resource?

Reply:
The templates are already packaged as embedded resources (after compression), I was wondering if we should also add these .tar.gz as content files within the nuget package additionally.

Reply:
Thanks for the clarification, since the Nugets are just for our use currently I am not sure there is much value in that change unless there was a specific use you were thinking of.  If we release the Nugets on Nuget.org then I think this would be a valuable change (though potentially should be in its own assembly).

Reply:
Yeah makes sense to consider when releasing the nuget publicly. Thanks